src/cmd/deploy_nsfs.sh

```diff
@@ -0,0 +1,416 @@
+#!/bin/bash
+
+### REQUIREMENTS ###
+# 1. NooBaa system installed on minikube
+# 2. kubectl cli installed
+# 3. aws cli installed
+# 4. noobaa cli installed
+
+# the results of this installation are - 
+# 1. noobaa nsfs exported bucket (RPC) - export of pre existing file system directory
+# 2. noobaa nsfs bucket (S3) - noobaa will create the file system directory during the request
+# see - https://github.com/noobaa/noobaa-core/wiki/NSFS-on-Kubernetes
+
+EXPORTED_BUCKET_NAME="exported-bucket"
+S3_BUCKET_NAME="nsfs-bucket-using-s3-client"
+NSFS_ACCOUNT_NAME="nsfs-account@noobaa.io"
+
+HELP="
+Help:
+    "deploy_nsfs" is a noobaa-core command deploys different resources in order to use namespace FS feature.
+    S3 bucket - is a NooBaa NSFS bucket that its underlying directory doesn't exist before the bucket creation request, NooBaa will create it during the bucket creation process.
+    Exported bucket (RPC) - is a NooBaa NSFS bucketthat its underlying directory exist before the bucket creation request, NooBaa will check the directory's existance during the bucket creation process.
+    For more information refer to the noobaa docs.
+    see - https://github.com/noobaa/noobaa-core/wiki/NSFS-on-Kubernetes
+";
+
+USAGE="
+Usage:
+    noobaa-core deploy_nsfs <deployment/cleanup>
+";
+
+ARGUMENTS="
+Arguments:
+    <deployment/cleanup>     NooBaa will deploy/cleanup a local storageclass, pvc, pv, nsfs namespacestore, RPC exported bucket, S3 bucket and an account for the S3 bucket
+";
+
+
+function print_usage() {
+    echo -e "$HELP";
+    echo -e "$USAGE";
+    echo -e "$ARGUMENTS";
+    exit 1;
+}
+
+
+# print_headline recieves 2 args - headline and explain and prints them
+function print_headline(){
+    headline_len=$((${#1} + 20))
+    printf '%0.s-' $(seq 1 $headline_len)
+    echo -e "\n--------- $1 ---------";
+    printf '%0.s-' $(seq 1 $headline_len)
+    echo -e "\n$2\n";
+
+}
+
+
+# print_done prints done
+function print_done(){
+    echo -e "done\n";
+}
+
+
+# set_endpoint_url sets and prints ENDPOINT 
+function set_endpoint_url(){
+ENDPOINT='https://localhost:10443';
+print_headline "S3 ENDPOINT" "NooBaa S3 endpoint is $ENDPOINT"
+print_done
+}
+
+
+# start_portforward starts port forwarding s3 service
+function start_portforward(){
+print_headline "S3 SERVICE PORT FORWARD START" "Port forwarding the S3 service"
+kubectl port-forward service/s3 10443:443 &
+print_done
+}
+
+
+# stop_portforward stops port forwarding s3 service
+function stop_portforward(){
+print_headline "S3 SERVICE PORT FORWARD STOP" "Stop port forwarding the S3 service"
+pkill -f "port-forward";
+print_done
+}
+
+
+# set_admin_credentials reads admin account, sets and prints admin_account_access_key and admin_account_secret_key
+function set_admin_credentials(){
+print_headline "ADMIN CREDENTIALS" ""
+admin_account=$(noobaa api account_api read_account '{"email":"admin@noobaa.io"}' -o json);
```

Comment:
I run `devenv.sh `so instead of `noobaa` in this case it's `nb`, we need to handle  this case.
Anyway, it didn't stop after failing in this step.

Reply:
I tried to add nb handling but it doesn't work and nb will not be recognized